src/store/fences/selector.ts

```diff
@@ -6,3 +9,24 @@ export type FencesProps = initialState;
 
 export const fencesSelector = (state: RootState) => state.fences;
 export const fencesAuthorizedStudiesSelector = (state: RootState) => state.fences.authorizedStudies;
+export const fencesAtLeastOneAuthentificationConnectedSelector = (state: RootState) =>
+  Object.keys(FENCE_NAMES).some((fenceKey) => {
+    const key = fenceKey as FENCE_NAMES;
+    if (state.fences[key] === undefined) {
+      return false;
+    }
+
+    return state.fences[key].status === FENCE_AUTHENTIFICATION_STATUS.connected;
+  });
+
+export const fencesAllAclsSelector = (state: RootState) => {
+  const acls: string[] = [];
```

Comment:
All good, but you can leverage filter and map for a more functional approach...up to you.

Reply:
Dans ce cas si, je dois merger les tableaux donc je pense que ça reste correct comme ça (et plus facile pour gérer le typage)